services/auth-server/Dockerfile

```
@@ -9,6 +9,7 @@ RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install -
 
 # Get all Python requirements in place and install them.
 COPY ./requirements.txt /orchest/services/auth-server/
+COPY ./lib /orchest/lib
```

The javascript library is already copied in in a later step and should not invalidate the cache for installing the Python requirements.

```suggestion
COPY ./lib/python /orchest/lib/python
```